From: Florin Dumitrascu <florin.dumitrascu@enea.com>
Date: Fri, 4 Mar 2016 17:19:41 +0100
Subject: [PATCH] Add cobbler-grub-aarch64 RPM reinstall to start.sh

Because /var/lib/cobbler is mounted as a volume, new cobbler aarch64
grub loader packaged as an RPM (cobbler-grub-aarch64) should be
reinstalled if its files are missing on Docker container start.
---
 docker/cobbler/start.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/docker/cobbler/start.sh b/docker/cobbler/start.sh
index 3c5d411..c1a359b 100644
--- a/docker/cobbler/start.sh
+++ b/docker/cobbler/start.sh
@@ -18,6 +18,9 @@ fi
 if rpm -V cobbler-web | grep -q missing; then
   yum reinstall -q -y cobbler-web
 fi
+if rpm -V cobbler-grub-aarch64 | grep -q missing; then
+  yum reinstall -q -y cobbler-grub-aarch64
+fi
 
 
 # Run puppet to apply custom config
-- 
1.9.1

